app/(company)/run-for-office/components/Blog.js

```diff
@@ -0,0 +1,43 @@
+'use client';
+import MaxWidth from '@shared/layouts/MaxWidth';
+import Pill from '@shared/buttons/Pill';
+import Image from 'next/image';
+
+export default function Blog({ articles }) {
+  return (
+    <MaxWidth>
+      <div className="flex items-center justify-center align-middle flex-col w-full mb-12">
+        <h3 className=" font-black text-4xl mt-10 mb-12">Our Blog</h3>
+        <div className="grid grid-cols-1 lg:grid-cols-3 gap-6">
```

Comment:
please keep the grid the same way I use it everywhere:
use `grid grid-cols-12` at the container
and in the children use
`col-span-12 lg:col-span-4`